CONTRIBUTING.md

```diff
@@ -1,18 +1,18 @@
-# Welcome to Scaffold-ETH 2 Contributing Guide
+# Welcome to Scaffold-Scroll Contributing Guide
 
-Thank you for investing your time in contributing to Scaffold-ETH 2!
+Thank you for investing your time in contributing to Scaffold-Scroll!
 
 This guide aims to provide an overview of the contribution workflow to help us make the contribution process effective for everyone involved.
 
 ## About the Project
 
-Scaffold-ETH 2 is a minimal and forkable repo providing builders with a starter kit to build decentralized applications on Ethereum.
+Scaffold-Scroll is a minimal and forkable repo providing builders with a starter kit to build decentralized applications on Ethereum.
 
 Read the [README](README.md) to get an overview of the project.
 
 ### Vision
 
-The goal of Scaffold-ETH 2 is to provide the primary building blocks for a decentralized application.
+The goal of Scaffold-Scroll is to provide the primary building blocks for a decentralized application.
```

Comment:
In line 47 of this file, I think we need to change scaffold-eth to the correct scaffold-scroll link for issues

Or we can also include a section about contributing to scaffold-eth via their issues and making similar changes to scaffold-scroll so that we stay up to date